incident_viewer_demo/src/App.js

```diff
@@ -4,29 +4,53 @@ import AutoWidth from 'react-auto-width';
 import logo from './demo-logo-center.png';
 import './App.scss';
 import LineChartWithCounts from './LineChart/LineChartWithCounts.jsx'
+import IspSelect from './IspSelect/IspSelect.jsx'
+import HelpTip from './HelpTip/HelpTip.jsx'
+
 import * as moment from 'moment';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './chart_support/assets/base.scss';
+import Icon from './Icon.jsx';
+import { colorsFor } from './chart_support/color';
 
 //////////////////////////////////////////////////////////////////////////
 // Handle all of the discombobulated variables that the chart takes in. //
 //////////////////////////////////////////////////////////////////////////
 
+// TODO: delete unused JSON before sending a pull request
+// var topClientIsps = require('./sample_data/all_isps.json'); // all ISPs for this query
+var ispsWithIncidents = require('./sample_data/isps_with_incidents.json'); // ISPs with incidents for this query
+
 var chartId = "providers-time-series"
-var colors = {naus_AS11486x: "rgb(125, 25, 125)", nauswaseattle_AS11398x: "rgb(69, 160, 58)", nauswaseattle_AS21928: "rgb(125, 66, 25)"}
 
-// Reading and loading JSON files with sample data of an incident
-var clientIspTimeSeriesData = require('./sample_data/client_isp_time_series_data.json');
-var annotationTimeSeries = require('./sample_data/annotation_time_series_data.json');
+// Reading and loading JSON files with sample ISP data
+var clientIspTimeSeriesData = require('./sample_data/newDemoData.json');
```

Comment:
why is this json file called newDemoData?

Reply:
I agree with Amy. This name should get changed since there is no mention of "oldDemoData" to contrast this with. It should just be named "demoData"